src/py/flwr/server/superlink/linkstate/utils.py

```diff
@@ -148,7 +155,7 @@ def context_from_bytes(context_bytes: bytes) -> Context:
 
 def make_node_unavailable_taskres(ref_taskins: TaskIns) -> TaskRes:
```

Comment:
@danieljanes Should we remove this? This was used to generate error `TaskRes` for the `TaskIns` of which the destination node is unavailable (offline).

Reply:
@panh99 yes, we can remove it.